src/main/java/net/minecraftforge/oredict/OreDictionary.java

```diff
@@ -226,6 +226,9 @@ private static void initVanillaEntries()
             registerOre("paneGlass",     Blocks.GLASS_PANE);
             registerOre("paneGlass",     new ItemStack(Blocks.STAINED_GLASS_PANE, 1, WILDCARD_VALUE));
             //paneGlass{Color} is added below with dyes
+            registerOre("wool",          Blocks.WOOL);
```

Comment:
This line shouldn't be here - all wool is "dyed" wool, there's no seperate "clear" wool block like glass has.

Reply:
Thanks for pointing that out.